lib/page-object/accessors.rb

```diff
@@ -1304,6 +1304,55 @@ def indexed_property (name, identifier_list)
     end
 
     #
+    # adds a method to return a specified class wrapping a generic Element object
+    #
+    # @example
+    #   page_section(:navigation_bar, :id => 'nav-bar')
+    #   # will generate 'navigation_bar' and 'navigation_bar?'
+    #
+    # @param [Symbol] the name used for the generated methods
+    # @param [Hash] identifier how we find an element.  You can use multiple parameters
+    #   by combining of any of the following except xpath.  The valid keys are:
+    #   * :class => Watir and Selenium
+    #   * :css => Selenium only
+    #   * :id => Watir and Selenium
+    #   * :index => Watir and Selenium
+    #   * :name => Watir and Selenium
+    #   * :xpath => Watir and Selenium
+    #
+    def page_section (name, section_class, identifier_list)
```

Comment:
I named the accessor methods page_section and page_sections because a section is a valid html tag already, but I didn't have a better word to describe what the classes represent. I think since they act a lot like page objects themselves, page section can be a good name for them.